HeterogeneousCore/CUDACore/python/SwitchProducerCUDA.py

```diff
@@ -1,6 +1,6 @@
 import FWCore.ParameterSet.Config as cms
 
-_cuda_enabled_cached = None
+_cuda_enabled_cached = True
```

Comment:
I'll have to think about this

Reply:
Please revert this change (and if the commit history will be cleaned up for other reasons, preferably from there as well).